torch/csrc/jit/passes/quantization.h

```diff
@@ -137,5 +138,11 @@ TORCH_API void FoldPrepackedWeightIntoModule(
     script::Module& module,
     const script::Module& linear_params_module,
     const script::Module& conv_params_module);
+
+/** Deduplicate multiple uses of the same module by
+ *  creating a new module for each use of the module
```

Comment:
It's probably worth adding more details in this comment: i.e. that this is recursive, it adds new submodule (a clone) to the module object **and** modifies the code of the type, what are the types of the created clones, what happens to the parameters/attributes of the cloned modules, etc.